src/amazon_invoice_downloader/cli/__init__.py

```diff
@@ -278,13 +317,15 @@ def run(playwright, args):
                         margin={"top": ".5in", "right": ".5in", "bottom": ".5in", "left": ".5in"},
                     )
                     invoice_page.close()
-
+    print("✅ Finished downloading invoices")
     # Close the browser
     context.close()
     browser.close()
 
 
 def amazon_invoice_downloader():
+    global DEBUG_MODE
```

Comment:
MUST: Would be great to avoid a global variable.  This program uses "docopt" library, so you it should automatically be passed in at L342 in your PR.  "run(playwright, args)"